docs/_pages/releases.md

```diff
@@ -3,8 +3,40 @@ title: "Releases"
 permalink: /releases
 layout: releases
 toc: true
-last_modified_at: 2019-12-30T15:59:57-04:00
+last_modified_at: 2020-05-28T08:40:00-07:00
 ---
+## [Release 0.5.3](https://github.com/apache/hudi/releases/tag/release-0.5.3) ([docs](/docs/0.5.3-quick-start-guide.html))
+
+### Download Information
+ * Source Release : [Apache Hudi 0.5.3 Source Release](https://downloads.apache.org/hudi/0.5.3/hudi-0.5.3.src.tgz) ([asc](https://downloads.apache.org/hudi/0.5.3/hudi-0.5.3.src.tgz.asc), [sha512](https://downloads.apache.org/hudi/0.5.3/hudi-0.5.3.src.tgz.sha512))
+ * Apache Hudi jars corresponding to this release is available [here](https://repository.apache.org/#nexus-search;quick~hudi)
+
+### Release Highlights
+ * Since this is a bug fix release, not a lot of new features as such. Will call out a few features and then will go over some of the improvements and some of the notable bug fixes. 
+ * `Features`: 
+ * Added support for `aliyun OSS` and `Presto MOR query` support to Hudi
+ 
+ * `Improvements`: 
+ * Improved write performance in creating Parquet DataSource after writes
```

Comment:
Can we make this user facing and understandable like the previous notes?  We need not list all the bugs/commit message here per se.. 

We also need a section on upgrading, which asks the user to consider the steps in upgrading from previous versions.. i.e if upgrading from 0.5.0 to 0.5.3 then the 0.5.1 and 0.5.2 upgrade steps need to be taken into consideration

Reply:
I didn't list all bugs as such. Probably need someone to assist me here. I can work with them to get the right ones. 

btw, wrt upgrading, there is nothing required to be done to upgrade. So, I did have a line saying the same, but @leesf suggested to remove if there aren't anything to be done. So, in my updates to the patch, I removed them.

Reply:
we need to definitely write something more user facing.. @nsivabalan I suggest you  take a stab at this first.. and then @bhasudha or @bvaradar  can help and review.. Essentially, we need to write it in the perspective of the users e.g "Incremental cleaning is turned on by default. This avoid full table listings for purposes of cleaning older versions of files" . Simply saying "Avoid unnecessary listings in incremental cleaning mode" is not very obvious.. 

On upgrading, we can discuss this more. but it does not hurt to priovide people with more context.. in case someone wants to upgrade from 0.5.0 to 0.5.3 directly.. then they will skip the steps needed for interim releases. we need to call  this out IMO

Reply:
@nsivabalan : How about starting with this release note : 

```
### Migration Guide for this release
 * This is a bug fix only release and no special migration steps needed when upgrading from 0.5.2. If you are upgrading from earlier releases “X”, please make sure you read the migration guide for each subsequent release between “X” and 0.5.3


### Release Highlights
 * Hudi now supports `aliyun OSS` storage service.
 * Embedded Timeline Server is enabled by default for both delta-streamer and spark datasource writes. This feature was in experimental mode before this release. Embedded Timeline Server caches file listing calls in Spark driver and serves them to Spark writer tasks. This reduces the number of file listings needed to be performed for each write.
 * Incremental Cleaning is enabled by default for both delta-streamer and spark datasource writes. This feature was also in experimental mode before this release. In the steady state, incremental cleaning avoids the costly step of scanning all partitions and instead uses Hudi metadata to find files to be cleaned up.
 * Delta-streamer config files can now be placed in different filesystem than actual data.
 * Hudi Hive Sync now supports tables partitioned by date type column.
 * Hudi Hive Sync now supports syncing directly via Hive MetaStore.
 * Other Writer Performance related fixes:
 ** DataSource Writer now avoids unnecessary loading of data after write.
 ** Hudi Writer now leverages spark parallelism when searching for existing files for writing new records.
 
### Raw Release Notes
   The raw release notes are available [here](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12322822&version=12348256)
```